modules/openxr/doc_classes/OpenXRHand.xml

```diff
@@ -18,6 +19,9 @@
 		<member name="motion_range" type="int" setter="set_motion_range" getter="get_motion_range" enum="OpenXRHand.MotionRange" default="0">
 			Set the motion range (if supported) limiting the hand motion.
 		</member>
+		<member name="skeleton_rig" type="int" setter="set_skeleton_rig" getter="get_skeleton_rig" enum="OpenXRHand.SkeletonRig" default="0">
+			Set the type of skeleton rig the [member hand_skeleton] is complaint with.
```

Comment:
```suggestion
			Set the type of skeleton rig the [member hand_skeleton] is compliant with.
```

Reply:
Whups - good catch. I rolled this in with the changes Bastiaan wanted for description clarification.